README.md

```diff
@@ -266,6 +266,22 @@ This script allows a certain degree of flexibility by allowing the following cus
 * Custom `config_override.php` - to provide additional settings that might be needed for your Sugar installation
 * Custom initialisation PHP script - allows to perform actions to the system after installation, leveraging Sugar Beans and any system logic (eg: load few test users, configure mail settings etc)
 
+#### jmeter/build.sh
+This script installs the jmeter components present on the [performance repository](https://github.com/sugarcrm/performance).
```

Comment:
Perhaps explain that this is a private repository and the user might not have access to it

Reply:
Sorry you explained that two lines below